src/restore.S

```diff
@@ -36,10 +36,13 @@ boot_decide_vector:
   bne t1, t2, read_incomplete # read incomplete goto bad_trap
 
   li s0, BOOT_FLAG_ADDR
+  la s1, place_of_restore_size
+  add s0, s0, s1
   ld t1, (s0) #load boot flag
   li t2, CPT_MAGIC_BUMBER
   beq t1, t2, restore_csr_vector
-  j payload_bin # not found restore flag, goto payload
+  li t0, 0x80000000
```

Comment:
Why is it changed to 0x8000_0000? The payload is linked to this gcpt.bin as described in the doc, which is expected to be called by `j payload_bin`.

If the payload and restorer are separated now, we need to update the doc and not link the payload in this bin.

Reply:
sorry about that, these repositories were supposed to be updated together, but currently LibcheckpointAlpha is the first to finish updating, so that's causing some confusion, I'll fix it as soon as I can, until then, please drop this commit